ArmPkg/Library/ArmFfaLib/ArmFfaStandaloneMmLib.c

```diff
@@ -68,5 +68,25 @@ ArmFfaStandaloneMmLibConstructor (
   IN EFI_MM_SYSTEM_TABLE  *MmSystemTable
   )
 {
-  return ArmFfaLibCommonInit ();
+  EFI_STATUS  Status;
+
+  Status = ArmFfaLibCommonInit ();
+  if (Status == EFI_UNSUPPORTED) {
+    /*
+     * EFI_UNSUPPORTED return from ArmFfaLibCommonInit() means
+     * FF-A interface isn't supported.
+     * However, It doesn't make failure of loading driver/library instance
+     * (i.e) ArmPkg's MmCommunication Dxe/PEI Driver uses as well as SpmMm.
+     * So If FF-A is not supported the the MmCommunication Dxe/PEI falls
+     * back to SpmMm.
+     * For this case, return EFI_SUCCESS.
+     */

```

Comment:
Please fix the grammar and capitalization on this comment block (and the double `the`). Also `i.e.` does not mean 'for example'.

Reply:
Agreed. Updated comment. Please let me know if any other suggestions.